inappmessaging/USERGUIDE.md

```diff
@@ -274,6 +312,15 @@ buildscript {
     }
 }
 ```
+</details>
+
```

Comment:
Instead of the details part below please change to:

```
### Other Issues
Rakuten developers experiencing any other problems should refer to the Troubleshooting Guide on the internal developer documentation portal.
```

Reply:
Changed